scripts/tasks/perf-test.js

```diff
@@ -76,16 +76,19 @@ async function runAvailableScenarios(page, componentCount, iterations, sampleSiz
   const perfNumbers = {};
 
   // Iterate through scenarios available
-  const scenarioDropdown = await page.$('.scenario');
-  let scenarioName = (await page.$eval('.scenario', dropdown => dropdown.textContent)).replace(/[^a-zA-Z\s]/g, '');
+  await page.evaluate(() => {
+    debugger;
```

Comment:
What does this do? Does this need to be checked in?

Reply:
Whoops, I'll take that out

Reply:
Done